### PR TITLE
fix(style guide): Improve "edit doc" instructions

### PR DESCRIPTION
**Pull request by austin-schaefer**
* Shuffle order of h2s in several style guide docs to surface more universally applicable info
* Simplify text for edit procedure to make it look less hard
* Reduce and simplify text related to licenses, scheduling edits, and private site. These procedures are uncommon, and giving them so much screen real estate makes things seem a bit hard

### CODE REVIEW

**Comment by austin-schaefer** [comment]
Rolled a d10 and it said @dbarnesbrownNR is my reviewer. :)

File: src/content/docs/style-guide/writing-guidelines/create-edit-content.mdx [hunk 1]
```diff
@@ -17,42 +17,20 @@ redirects:
...
-1. Find a doc you'd like to edit.
-2. Click **Edit file** on the top corner of the right nav.
-3. In the GitHub file, click on the **pencil** icon within the code block on the top right corner to access the [markdown](https://en.wikipedia.org/wiki/Markdown) editor.
-4. Make your changes.
-5. Write a meaningful commit message and description, then commit your change by creating a new branch.
-6. Follow the prompts to submit your pull request. Please fill in as many details as possible in the provided template.
-7. And you're done!
-
-A member of the Tech Docs team will let you know when we've published your changes.
+1. On the docs site, navigate to the doc you'd like to edit..
```
Comment (urbiz-nr) [inline src/content/docs/style-guide/writing-guidelines/create-edit-content.mdx:26]: There's two points at the end of the sentence.

File: src/content/docs/style-guide/writing-guidelines/create-edit-content.mdx [hunk 2]
```diff
@@ -17,42 +17,20 @@ redirects:
...
+2. Click **Edit page** on the top corner of the right nav.
+3. A GitHub page will open with the source of the doc. Click the  **pencil** icon in the top right.
```
Comment (urbiz-nr) [inline src/content/docs/style-guide/writing-guidelines/create-edit-content.mdx:28]: Are there two spaces before **pencil**?

File: src/content/docs/style-guide/writing-guidelines/create-edit-content.mdx [hunk 3]
```diff
@@ -17,42 +17,20 @@ redirects:
...
+4. Make your edits (don't worry too much about formatting or grammar, we're happy to worry about that).
```
Comment (urbiz-nr) [inline src/content/docs/style-guide/writing-guidelines/create-edit-content.mdx:29]: I wouldn't use "worry" twice. How about "We're happy to take care of that."

**Review by urbiz-nr — DISMISSED** [review]
Nice update! Please check my comments.

**Comment by dbarnesbrownNR** [comment]
@austin-schaefer this is showing up fine on preview but that errant extra period @urbiz-nr caught is still there and I agree re: the use of "worry" twice in step 4 of **Edit a doc**.

File: src/content/docs/style-guide/writing-guidelines/create-edit-content.mdx (hunk 1 above)
Comment (austin-schaefer) [reply src/content/docs/style-guide/writing-guidelines/create-edit-content.mdx:26]: Yikes, good catch!

File: src/content/docs/style-guide/writing-guidelines/create-edit-content.mdx (hunk 2 above)
Comment (austin-schaefer) [reply src/content/docs/style-guide/writing-guidelines/create-edit-content.mdx:28]: Sharp eye indeed, thanks

File: src/content/docs/style-guide/writing-guidelines/create-edit-content.mdx (hunk 3 above)
Comment (austin-schaefer) [reply src/content/docs/style-guide/writing-guidelines/create-edit-content.mdx:29]: 👍

**Review by dbarnesbrownNR — APPROVED** [review]
Changes look good, all set!

**Pull request merged**